docs/topics/api/reviewers.rst

```diff
@@ -10,7 +10,8 @@ Reviewers
     See :ref:`the API versions available<api-versions-list>` for alternatives
     if you need stability.
     The only authentication method available at
-    the moment is :ref:`the internal one<api-auth-internal>`.
+    the moment is :ref:`the internal one<api-auth-internal>`, except for
```

Comment:
This is a confusing sentence. There are two auth methods, but the sentence starts with "the only one available" can you rephrase this?

Reply:
Maybe 

```
The only authentication method available for these APIs is
:ref:`the internal one<api-auth-internal>`, except for the
:ref:`validation results<reviewers-validation>` endpoint, which allows both
internal and :ref:`external auth<api-auth>`.
```

? Ultimately internal auth remains the only auth mechanism available for the others APIs described in that doc, validation is the exception...